### ISSUE TITLE
Icons are not aligned correctly

### ISSUE THREAD

**Issue by acidjazz** [Wikiki/bulma-timeline#24]
bulma 0.6.2 and fontawesome v5.0.11

source
```html
<div class="timeline-marker is-icon is-success">
  <i class="fas fa-plus"></i>
</div>
```

result

![image](https://user-images.githubusercontent.com/967369/39851891-3e4f6284-53df-11e8-983e-2c341715960f.png)

**Comment by acidjazz** [comment]
this probably doesn't help but here's a hack in stylus i've written as a temp fix 

```stylus

.timeline-marker.is-icon
  padding: 0.55rem !important
  padding-left: 0.65rem !important
  border .0em solid #fefefe !important
  width 2em !important
  height 2em !important
  left -1em !important
.timeline.is-centered .timeline-item:nth-of-type(2n) .timeline-marker
  left: auto !important
  right: -1em !important
```

**Comment by Wikiki** [comment]
Fixed feab6be
